### PR TITLE
Release/0.1.0

### PR DESCRIPTION
**Pull request by AlekseiiKanaiev**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1 @@
+<h1>Hello, World!</h1>
```
Comment (the-daria) [inline index.html:1]: Please add more text to test your understanding of comments.

Comment (AlekseiiKanaiev) [reply index.html]: Added more text to make clear, that I understood coment

File: index.html [hunk 2]
```diff
@@ -1 +1,2 @@
 <h1>Hello, World!</h1>
+<p>More text</p>
```
Comment (AlekseiiKanaiev) [inline index.html]: Added more text to make clear, that I understood coment

File: index.html [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Hello, World!</h1>
+<p>More text</p>
```
Comment (AlekseiiKanaiev) [inline index.html:2]: Added more text to make clear, that I understood coment

**Pull request merged**